### PR TITLE
doc(tableWidget) How use new properties for Table and Data table widget

### PR DESCRIPTION
**Pull request by benjaminParisel**
Covers [UID-205](https://bonitasoft.atlassian.net/browse/UID-205)

### CODE REVIEW

**Review by pabloalonso — APPROVED** [review]
LGTM

File: md/widgets.md [hunk 1]
```diff
@@ -247,17 +247,64 @@ Use a display widget for information that the user can read on a page. This incl
...
+
+#### Customize look and feel
+
+Styling properties are available:
+
+* Zebra-Striping to set a different background color on even and odd lines
+* Condensed to display cells without spacing
+* Bordered to display cells with border
+
+You can customize default display for zebra-styling or bordered. Here an example to explain you how to customize default zebra styling.
```
Comment (jeromecambon) [inline md/widgets.md]:
```suggestion
You can customize default display for zebra-striping or bordered. Here is an example to explain you how to customize default zebra styling. 
```

File: md/widgets.md [hunk 2]
```diff
@@ -247,17 +247,64 @@ Use a display widget for information that the user can read on a page. This incl
...
 To get the data from a selected row, bind **selected row** to a variable.
 
-Note: table widget only supports text values. HTML content passed in table data will not be rendered.
+With 7.10 update, you can now display html in header or data content when `Interpret HTML` property is set to true. HTML sanitization is called before rendering to keep only safe HTML.
+
+#### Customize look and feel
+
+Styling properties are available:
+
+* Zebra-Striping to set a different background color on even and odd lines
```
Comment (jeromecambon) [inline md/widgets.md:264]:
```suggestion
* Zebra-striping to set a different background color on even and odd lines
```

File: md/widgets.md [hunk 3]
```diff
@@ -247,17 +247,64 @@ Use a display widget for information that the user can read on a page. This incl
...
+* Condensed to display cells without spacing
+* Bordered to display cells with border
+
+You can customize default display for zebra-styling or bordered. Here an example to explain you how to customize default zebra styling. 
+
+This styling can be done in the theme.css file in Bonita Studio part to customize **all** the tables. You need to use this following css selector:
```
Comment (jeromecambon) [inline md/widgets.md]:
```suggestion
This styling can be done in the theme.css file in Bonita Studio part to customize **all** the tables. You need to use the following css selector: 
```

File: md/widgets.md [hunk 4]
```diff
@@ -247,17 +247,64 @@ Use a display widget for information that the user can read on a page. This incl
...
+    }
+        
+    .zebra-custom .table-striped > tbody > tr:nth-of-type(even) {
+      // Put you custome styling here
+      background-color: black;
+    }
+    ```
+
+
+Note: In embedded Table widget documentation you can found an example for bordered custom style.
```
Comment (jeromecambon) [inline md/widgets.md:302]: I think it would be better to put the example here as well, and avoid to refer to embedded doc.

**Review by jeromecambon — CHANGES_REQUESTED** [review]

**Pull request merged**